package.json

```diff
@@ -43,8 +45,6 @@
     "katex": "^0.12.0",
     "lodash": "^4.17.19",
     "mini-css-extract-plugin": "^0.11.0",
-    "moment": "^2.24.0",
```

Comment:
This needs a major `PROVISION_VERSION` bump in `version.py`.

Reply:
Bumped it to <code>112</code> in <a href = 'https://github.com/zulip/zulip/blob/master/version.py#L47'>version.py</a> .